eclipse/sw360
# fix(ui): Changes in External urls in Project are not registered in Moderation Request. Closed Moderation Request doesnot show Proposed changes

> Please provide a summary of your changes here.
> * Which issue is this pull request belonging to and how is it solving it? (*Refer to issue here*)
> * Did you add or update any new dependencies that are required for your change?

Issue: closes #1343 

### How To Test?
> Create Moderation Request for project with changes in External URl - single/multi value.
>   - Validate Moderation Request data is correct.
>   -  Accept Moderation Request.

> Check Open/Closed Moderation Request for Project, Component, Release, License, User. should contain proposed changes. Also need to test Accept/Reject.

> Have you implemented any additional tests?

### Checklist
Must:

Signed-off-by: Jaideep Palit <jaideep.palit@siemens.com>
